core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -8131,6 +8131,7 @@ public static final class Name {
     public static final String PROXY_WEB_PORT = "alluxio.proxy.web.port";
     public static final String PROXY_AUDIT_LOGGING_ENABLED =
         "alluxio.proxy.audit.logging.enabled";
+    public static final String S3_UPLOADS_ID_XATTR_KEY = "s3_uploads_mulitpartupload_id";
```

Comment:
why is this property key string (both the variable name and the string value) formatted differently from the others? is it actually a configurable property key or an identifier for something else entirely?

Reply:
so it is a S3 constants, these constants are usually pit in S3Constants class, but it needs to be used on master code, and master doesnt have dependency on s3 module, therefore put it in a common place such as PropertyKey